CHANGELOG.md

```diff
@@ -95,6 +95,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Prevent deletion of snapshots that are backing searchable snapshot indexes ([#5069](https://github.com/opensearch-project/OpenSearch/pull/5069))
 - Add max_shard_size parameter for shrink API ([#5229](https://github.com/opensearch-project/OpenSearch/pull/5229))
 - Added support to apply index create block ([#4603](https://github.com/opensearch-project/OpenSearch/issues/4603))
+- Support request level durability for remote-backed indexes ([#5671](https://github.com/opensearch-project/OpenSearch/pull/5671))
```

Comment:
Super nitpick here, but `https://github.com/opensearch-project/OpenSearch/pull/5671` works but redirects to `https://github.com/opensearch-project/OpenSearch/issues/5671`. Can you link directly to the issue?

Reply:
Yes, done. Nice catch :)